crates/sp-consensus-subspace/src/digests.rs

```diff
@@ -44,11 +43,21 @@ pub struct Solution {
     pub tag: Tag,
 }
 
-impl Solution {
+impl<AccountId> Solution<AccountId>
+where
+    AccountId: Parameter
+        + Member
+        + MaybeSerializeDeserialize
+        + Debug
+        + MaybeDisplay
+        + Ord
+        + Default
+        + MaxEncodedLen,
```

Comment:
Just `Default` should be fine.

```suggestion
    AccountId: Default
```

Reply:
Good catch, I made it way more complex than it should have been